canonical-web-and-design/canonical.com
# 1658 link game to jobs page

## Done
- Add `/careers/results` page
- Link `careers/start` to `careers/results`


## QA
- Run the website locally using `./run`
- Go to [/careers/start](http://0.0.0.0:8002/careers/start) and select 5 skills (you should use at least one of these "Designer", "Problem Solver", "Technologist" to be able to find any jobs) and press `Submit choices`
- Check the page you get has a job, or none


## Issue
Fixes https://github.com/canonical-web-and-design/web-squad/issues/1658
